### PR TITLE
tool: Cleanup building app

### PR DESCRIPTION
**Pull request by provokateurin**
Paths that need to be mounted for building the app weren't included.

From https://github.com/provokateurin/nextcloud-neon/pull/372

### CODE REVIEW

File: tool/build/linux.sh [hunk 1]
```diff
@@ -13,4 +13,4 @@ trap cleanup EXIT
 
 cd /src/packages/app
 # shellcheck disable=SC2086
-flutter build linux ${BUILD_ARGS[*]}
\ No newline at end of file
+flutter build linux ${BUILD_ARGS[*]}
```
Comment (Leptopoda) [inline tool/build/linux.sh:16]: shouldn't we use fvm here?

File: tool/build-app.sh [hunk 2]
```diff
@@ -44,12 +44,8 @@ if [[ "$target" == "linux/arm64" ]] || [[ "$target" == "linux/amd64" ]]; then
   -f "tool/build/Dockerfile.$os" \
   ./tool/build
 
-  paths=()
-  for path in packages/{app,file_icons,nextcloud,settings,sort_box} packages/neon/*; do
-    paths+=("$path/lib" "$path/pubspec.yaml")
-  done
   run_args=()
-  for path in ${paths[*]} "packages/file_icons/fonts" packages/neon/neon{,_files,_news,_notes,_notifications}/assets "packages/app/build" "packages/app/linux"; do
+  for path in packages/{app,file_icons,nextcloud,settings,sort_box}/{lib,pubspec.yaml} packages/neon/*/{assets,lib,pubspec.yaml,pubspec_overrides.yaml} packages/file_icons/fonts packages/nextcloud/pubspec_overrides.yaml packages/app/{pubspec_overrides.yaml,assets,build,linux}; do
```
[the remark this answers is not in the document]
Comment (provokateurin) [reply tool/build-app.sh:48]: I was thinking about it, but the path mapping is really annoying. It is possible to only list packages that contain a certain file or directory, but you have to append the file or directory name yourself which is quite complex to do in bash (at least more than I'm willing to do)

[the remark this answers is not in the document]
Comment (Leptopoda) [reply tool/build-app.sh:48]: I mean specifying a custom command in the melos.yaml file with a specified regex including/excluding whatever we need.

Than the bash script would not need to specify any dir at all.

**Comment by provokateurin** [comment]
Meh this is broken. I think I tested it when I made the changes initially, but apparently I have missed something.

**Pull request merged**